.gitignore

```diff
@@ -50,3 +50,4 @@ install/kubernetes/istio.yaml
 samples/bookinfo/consul/bookinfo.sidecars.yaml
 samples/bookinfo/eureka/bookinfo.sidecars.yaml
 istio.VERSION
+/vendor/*
```

Comment:
not sure we should do that, some PR do need to update vendor - but maybe indeed this is safer to avoid most accident